dev/docker-compose.yml

```diff
@@ -9,27 +9,27 @@ services:
       - POSTGRES_DB=zimfarm
       - POSTGRES_USER=zimfarm
       - POSTGRES_PASSWORD=zimpass
-  zimfarm_api:
-    image: ghcr.io/openzim/zimfarm-dispatcher:latest
+  zimfarm-api:
+    image: docker.io/library/local-zimfarm-api
```

Comment:
what's this? I guess it's a _local/fake_ ID but why not using `build` and why reusing a publicly addressable name?

Reply:
This was not supposed to make it to the PR 🤣